pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -35,3 +48,31 @@ def toolchain_prefix(self) -> str:
             return f'riscv32-{self.target}-elf-'
         else:
             raise ValueError(f'Unknown target: {self.target}')
+
+    def image_info(self, path_to_file):
+        """
+        get hash256 of app
+
+        :param: path: path to file
+        :return: sha256 appended to app
+        """
+
+        old_stdout = sys.stdout
+        new_stdout = io.StringIO()
+        sys.stdout = new_stdout
```

Comment:
If an exception happens between this line and `sys.stdout = old_stdout` below, sys.stdout won't be restored